### PR TITLE
Sliding Pane Additions

### PR DESCRIPTION
**Pull request by CharlieLinx**
-Option to not close pane when click outside pane
-Example of applying Custom Styles to the Right pane including changing direction of close arrow.

### CODE REVIEW

File: src/index.styl [hunk 1]
```diff
@@ -102,3 +106,42 @@ $pane-header-height = 64px
     &__subtitle
         font-size 12px
         margin-top 2px
+
+    &__overlay_custom
```
Comment (DimitryDushkin) [inline src/index.styl]: is it needed in main source code?

File: src/index.js [hunk 2]
```diff
@@ -7,17 +7,21 @@ import './index.styl';
...
-    subtitle,
-    onRequestClose,
-    onAfterOpen,
-    children,
-    className,
-    overlayClassName,
-    from = 'right',
-    width
-}) {
+                                             isOpen,
```
[the remark this answers is not in the document]
Comment (CharlieLinx) [reply src/index.js]: Done

File: src/index.styl (hunk 1 above)
Comment (CharlieLinx) [reply src/index.styl]: I have moved this out of the main source into its own css file.

**Comment by CharlieLinx** [comment]
Hi Dimitry, why did you close my PR?

**Comment by DimitryDushkin** [comment]
Hey @CharlieLinx ! I totally refactored library, moved to TS, stylus -> CSS, Webpack -> Rollup. So there are a lot of merge conflicts. Sorry for that.

And honestly I still didn't test compatibility of new setup, so there is no new release yet.

If you are still willing to make this PR, I have to ask you to remake it within new setup. Sorry again.

**Pull request closed without merging**